### ISSUE TITLE
inner_join in odbc missing "AS" in TBL_RIGHT

### ISSUE THREAD

**Issue by hdelrio** [tidyverse/dplyr#3364]
When trying to make an inner_join between two tables using an odbc connection it fails as can be seen in the example below I have **bolded** the part which creates the failure

> library(DBI)
> con <- dbConnect(odbc::odbc(), "NFL")
> offense_db = tbl(con,'offense')
> player_db = tbl(con, 'player')
> offense_db %>% inner_join(player_db %>% filter(pos1=='QB'), by='player')
Error in new_result(connection@ptr, statement) : 
  nanodbc/nanodbc.cpp:1344: HY000: [SQLite]no such column: TBL_RIGHT.fname (1) 
> offense_db %>% inner_join(player_db %>% filter(pos1=='QB'), by='player') %>% show_query()
<SQL>
SELECT "TBL_LEFT"."uid" AS "uid", "TBL_LEFT"."gid" AS "gid", "TBL_LEFT"."player" AS "player", "TBL_LEFT"."pa" AS "pa", "TBL_LEFT"."pc" AS "pc", "TBL_LEFT"."py" AS "py", "TBL_LEFT"."ints" AS "ints", "TBL_LEFT"."tdp" AS "tdp", "TBL_LEFT"."ra" AS "ra", "TBL_LEFT"."sra" AS "sra", "TBL_LEFT"."ry" AS "ry", "TBL_LEFT"."tdr" AS "tdr", "TBL_LEFT"."trg" AS "trg", "TBL_LEFT"."rec" AS "rec", "TBL_LEFT"."recy" AS "recy", "TBL_LEFT"."tdrec" AS "tdrec", "TBL_LEFT"."ret" AS "ret", "TBL_LEFT"."rety" AS "rety", "TBL_LEFT"."tdret" AS "tdret", "TBL_LEFT"."fuml" AS "fuml", "TBL_LEFT"."peny" AS "peny", "TBL_LEFT"."conv" AS "conv", "TBL_LEFT"."fp" AS "fp", "TBL_LEFT"."fp2" AS "fp2", "TBL_LEFT"."fp3" AS "fp3", "TBL_LEFT"."game" AS "game", "TBL_LEFT"."seas" AS "seas", "TBL_LEFT"."year" AS "year", "TBL_LEFT"."team" AS "team", "TBL_LEFT"."posd" AS "posd.x", "TBL_LEFT"."jnum" AS "jnum.x", "TBL_LEFT"."dcp" AS "dcp.x", "TBL_LEFT"."nflid" AS "nflid.x", "TBL_RIGHT"."fname" AS "fname", "TBL_RIGHT"."lname" AS "lname", "TBL_RIGHT"."pname" AS "pname", "TBL_RIGHT"."pos1" AS "pos1", "TBL_RIGHT"."pos2" AS "pos2", "TBL_RIGHT"."height" AS "height", "TBL_RIGHT"."weight" AS "weight", "TBL_RIGHT"."dob" AS "dob", "TBL_RIGHT"."forty" AS "forty", "TBL_RIGHT"."bench" AS "bench", "TBL_RIGHT"."vertical" AS "vertical", "TBL_RIGHT"."broad" AS "broad", "TBL_RIGHT"."shuttle" AS "shuttle", "TBL_RIGHT"."cone" AS "cone", "TBL_RIGHT"."arm" AS "arm", "TBL_RIGHT"."hand" AS "hand", "TBL_RIGHT"."dpos" AS "dpos", "TBL_RIGHT"."col" AS "col", "TBL_RIGHT"."dv" AS "dv", "TBL_RIGHT"."start" AS "start", "TBL_RIGHT"."cteam" AS "cteam", "TBL_RIGHT"."posd" AS "posd.y", "TBL_RIGHT"."jnum" AS "jnum.y", "TBL_RIGHT"."dcp" AS "dcp.y", "TBL_RIGHT"."nflid" AS "nflid.y", "TBL_RIGHT"."pimg" AS "pimg"
  FROM "offense" AS "TBL_LEFT"
  INNER JOIN (SELECT *
FROM "player"

**WHERE ("pos1" = 'QB')) "TBL_RIGHT"**

  ON ("TBL_LEFT"."player" = "TBL_RIGHT"."player")

When the same query is made with a DBI connection, the inner_join functions perfect as can be seen in the example below, I have **bolded** the corresponding part

> con <- DBI::dbConnect(RSQLite::SQLite(), dbname = "data/nfl.sqlite")
> offense_db = tbl(con,'offense')
> player_db = tbl(con, 'player')
> offense_db %>% inner_join(player_db %>% filter(pos1=='QB'), by='player') %>% show_query()
<SQL>
SELECT `TBL_LEFT`.`uid` AS `uid`, `TBL_LEFT`.`gid` AS `gid`, `TBL_LEFT`.`player` AS `player`, `TBL_LEFT`.`pa` AS `pa`, `TBL_LEFT`.`pc` AS `pc`, `TBL_LEFT`.`py` AS `py`, `TBL_LEFT`.`ints` AS `ints`, `TBL_LEFT`.`tdp` AS `tdp`, `TBL_LEFT`.`ra` AS `ra`, `TBL_LEFT`.`sra` AS `sra`, `TBL_LEFT`.`ry` AS `ry`, `TBL_LEFT`.`tdr` AS `tdr`, `TBL_LEFT`.`trg` AS `trg`, `TBL_LEFT`.`rec` AS `rec`, `TBL_LEFT`.`recy` AS `recy`, `TBL_LEFT`.`tdrec` AS `tdrec`, `TBL_LEFT`.`ret` AS `ret`, `TBL_LEFT`.`rety` AS `rety`, `TBL_LEFT`.`tdret` AS `tdret`, `TBL_LEFT`.`fuml` AS `fuml`, `TBL_LEFT`.`peny` AS `peny`, `TBL_LEFT`.`conv` AS `conv`, `TBL_LEFT`.`fp` AS `fp`, `TBL_LEFT`.`fp2` AS `fp2`, `TBL_LEFT`.`fp3` AS `fp3`, `TBL_LEFT`.`game` AS `game`, `TBL_LEFT`.`seas` AS `seas`, `TBL_LEFT`.`year` AS `year`, `TBL_LEFT`.`team` AS `team`, `TBL_LEFT`.`posd` AS `posd.x`, `TBL_LEFT`.`jnum` AS `jnum.x`, `TBL_LEFT`.`dcp` AS `dcp.x`, `TBL_LEFT`.`nflid` AS `nflid.x`, `TBL_RIGHT`.`fname` AS `fname`, `TBL_RIGHT`.`lname` AS `lname`, `TBL_RIGHT`.`pname` AS `pname`, `TBL_RIGHT`.`pos1` AS `pos1`, `TBL_RIGHT`.`pos2` AS `pos2`, `TBL_RIGHT`.`height` AS `height`, `TBL_RIGHT`.`weight` AS `weight`, `TBL_RIGHT`.`dob` AS `dob`, `TBL_RIGHT`.`forty` AS `forty`, `TBL_RIGHT`.`bench` AS `bench`, `TBL_RIGHT`.`vertical` AS `vertical`, `TBL_RIGHT`.`broad` AS `broad`, `TBL_RIGHT`.`shuttle` AS `shuttle`, `TBL_RIGHT`.`cone` AS `cone`, `TBL_RIGHT`.`arm` AS `arm`, `TBL_RIGHT`.`hand` AS `hand`, `TBL_RIGHT`.`dpos` AS `dpos`, `TBL_RIGHT`.`col` AS `col`, `TBL_RIGHT`.`dv` AS `dv`, `TBL_RIGHT`.`start` AS `start`, `TBL_RIGHT`.`cteam` AS `cteam`, `TBL_RIGHT`.`posd` AS `posd.y`, `TBL_RIGHT`.`jnum` AS `jnum.y`, `TBL_RIGHT`.`dcp` AS `dcp.y`, `TBL_RIGHT`.`nflid` AS `nflid.y`, `TBL_RIGHT`.`pimg` AS `pimg`
  FROM `offense` AS `TBL_LEFT`
  INNER JOIN (SELECT *
FROM `player`

**WHERE ("pos1" = "QB")) AS "TBL_RIGHT"**

  ON (`TBL_LEFT`.`player` = `TBL_RIGHT`.`player`)

The problem is simple to solve as can be seen from the above two examples, the **bolded** part, I have to change the quotes in the **bolded** part of the second example so the markdown would look similar in both cases

**Comment by edgararuiz** [comment]
Hi, what is the database type for the **NFL** data source referenced to in the first examp

**Comment by hdelrio** [comment]
I’m addressing the same database in both cases, that is why it is a driver problem. In the first case I’m connecting using odbcsqlite in the second case I’m using DBI RSqlite. To fix the problem it has to be looked into the implementation of inner_join for the odbcsqlite driver (it might be the same for any odbc interface, I don’t have a way to check that) whomever implemented the SQL generating query missed the “AS” part

**Comment by edgararuiz** [comment]
Ok, so the issue is that the first connection is not being recognized as SQLite so it doesn't use a customization for that database inside `dbplyr`: https://github.com/tidyverse/dbplyr/blob/ac41874070e063b939726cbb999ebe800dab2a27/R/db-sqlite.r#L51-L61

To confirm, please use `class(con)` on the first example.  The result should be different from `SQLiteConnection`

**Comment by hdelrio** [comment]
This is clean environment for the first connection type (ODBC driver for Sqlite database)

> library(dplyr)

Attaching package: ‘dplyr’

The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

> library(odbc)
Warning message:
package ‘odbc’ was built under R version 3.4.3
> con <- dbConnect(odbc::odbc(), "NFL")
> class(con)
[1] "SQLite"
attr(,"package")
[1] ".GlobalEnv"
> offense_db = tbl(con,'offense')
> player_db = tbl(con, 'player')
> offense_db %>% inner_join(player_db %>% filter(pos1=='QB'), by='player')
Error in new_result(connection@ptr, statement) :
  nanodbc/nanodbc.cpp:1344: HY000: [SQLite]no such column: TBL_RIGHT.fname (1)
>


This is a clean environment for the second connection type (DBI RSqlite using the same database)

> library(dplyr)

Attaching package: ‘dplyr’

The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

> con <- DBI::dbConnect(RSQLite::SQLite(), dbname = "data/nfl.sqlite")
> class(con)
[1] "SQLiteConnection"
attr(,"package")
[1] "RSQLite"
> offense_db = tbl(con,'offense')
> player_db = tbl(con, 'player')
> offense_db %>% inner_join(player_db %>% filter(pos1=='QB'), by='player')
# Source:   lazy query [?? x 59]
# Database: sqlite 3.19.3 [/Users/heberto/Dropbox/Documents/Development/R/NFL/data/nfl.sqlite]
     uid   gid player     pa    pc    py  ints   tdp    ra   sra    ry   tdr   trg   rec  recy tdrec   ret
   <int> <int> <chr>   <int> <int> <int> <int> <int> <int> <int> <int> <int> <int> <int> <int> <int> <int>
 1     3     1 JG-0600    36    23   252     1     3     2     1    22     0     0     0     0     0     0
 2     8     1 CC-1400    31    16   264     0     2     4     1    12     0     1     1    -4     0     0
 3    20     2 MB-4500    34    24   301     0     1     1     1     5     0     0     0     0     0     0
 4    25     2 TC-2600    28    19   153     0     1     2     1    32     0     0     0     0     0     0
 5    40     2 JM-1100     0     0     0     0     0     3     0   - 5     0     0     0     0     0     0
 6    41     3 DM-2200    28    16   130     2     1     5     4    29     1     0     0     0     0     0
 7    47     3 TA-0300     5     0     0     1     0     0     0     0     0     0     0     0     0     0
 8    58     3 RC-2800    27    13   130     1     1     1     1     5     0     0     0     0     0     0
 9    64     3 KD-0600     1     0     0     1     0     0     0     0     0     0     0     0     0     0
10    67     4 BF-0100    34    14   152     1     1     2     2    13     0     0     0     0     0     0
# ... with more rows, and 42 more variables: rety <int>, tdret <int>, fuml <int>, peny <int>, conv <int>,
#   fp <dbl>, fp2 <dbl>, fp3 <dbl>, game <int>, seas <int>, year <int>, team <chr>, posd.x <chr>,
#   jnum.x <int>, dcp.x <int>, nflid.x <chr>, fname <chr>, lname <chr>, pname <chr>, pos1 <chr>, pos2 <chr>,
#   height <int>, weight <int>, dob <chr>, forty <dbl>, bench <int>, vertical <int>, broad <int>,
#   shuttle <dbl>, cone <dbl>, arm <int>, hand <int>, dpos <int>, col <chr>, dv <chr>, start <int>,
#   cteam <chr>, posd.y <chr>, jnum.y <int>, dcp.y <int>, nflid.y <chr>, pimg <chr>
>

--
Heberto del Rio
Mathematics and Computer Science
Barry University


On February 21, 2018 at 5:23:32 PM, Edgar Ruiz (notifications@github.com<mailto:notifications@github.com>) wrote:

Ok, so the issue is that the first connection is not being recognized as SQLite so it doesn't use a customization for that database inside dbplyr: https://github.com/tidyverse/dbplyr/blob/ac41874070e063b939726cbb999ebe800dab2a27/R/db-sqlite.r#L51-L61<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Ftidyverse%2Fdbplyr%2Fblob%2Fac41874070e063b939726cbb999ebe800dab2a27%2FR%2Fdb-sqlite.r%23L51-L61&data=02%7C01%7Chdelrio%40barry.edu%7Ca63a7a5b37114a45b13808d57979bbe6%7C7549241466ff4a7ca7f16c6787413d30%7C0%7C0%7C636548486120382591&sdata=TFnRqzUhPlPY4oZfi%2FffhtjyXcGhDEMzNzlyIGyq1NQ%3D&reserved=0>

To confirm, please use class(con) on the first example. The result should be different from SQLiteConnection

**Comment by edgararuiz** [comment]
Yup, this confirms it.  The connection call for the initial connection is `SQLite`, and the `dbplyr` is looking for `SQLiteConnection` to use the correct customization

**Comment by hdelrio** [comment]
Is there any way to fix it?

--
Heberto del Rio
Mathematics and Computer Science
Barry University

On February 21, 2018 at 5:35:13 PM, Edgar Ruiz (notifications@github.com<mailto:notifications@github.com>) wrote:

Yup, this confirms it. The connection call for the initial connection is SQLite, and the dbplyr is looking for SQLiteConnection to use the correct customization

**Comment by edgararuiz** [comment]
There's a workaround you can use to point your current connection to the correct translation:

`sql_subquery.SQLite <- sql_subquery.SQLiteConnection`

**Comment by hdelrio** [comment]
I try it, unsuccessfully :(

> library(dplyr)

Attaching package: ‘dplyr’

The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

> library(odbc)
Warning message:
package ‘odbc’ was built under R version 3.4.3
> con <- dbConnect(odbc::odbc(), "NFL")
> sql_subquery.SQLite = sql_subquery.SQLiteConnection
Error: object 'sql_subquery.SQLiteConnection' not found
>

--
Heberto del Rio
Mathematics and Computer Science
Barry University


On February 21, 2018 at 5:55:12 PM, Edgar Ruiz (notifications@github.com<mailto:notifications@github.com>) wrote:

There's a workaround you can use to point your current connection to the correct translation:

sql_subquery.SQLite <- sql_subquery.SQLiteConnection

**Comment by edgararuiz** [comment]
Oh, that function is in `dbplyr`

**Comment by hdelrio** [comment]
Nope, sql_subquery is a function in dplyr, this function has several parameters, the first one is a connection, at least that is what the documentation says.

On the other hand in dbplyr in the db-sqlite.R file there is a function whose name is sql_subquery, but then what is the “.SQLite” or the “.SQLiteConnection” coming from, that part is not documented in any of dplyr or dbplyr
Database and SQL generics.
Description

The sql_ generics are used to build the different types of SQL queries. The default implementations in dbplyr generates ANSI 92 compliant SQL. The db_ generics execute actions on the database. The default implementations in dbplyr typically just call the standard DBI S4 method.

Usage

db_desc(x)

sql_translate_env(con)

db_list_tables(con)

db_has_table(con, table)

db_data_type(con, fields)

db_save_query(con, sql, name, temporary = TRUE, ...)

db_begin(con, ...)

db_commit(con, ...)

db_rollback(con, ...)

db_write_table(con, table, types, values, temporary = FALSE, ...)

db_create_table(con, table, types, temporary = FALSE, ...)

db_insert_into(con, table, values, ...)

db_create_indexes(con, table, indexes = NULL, unique = FALSE, ...)

db_create_index(con, table, columns, name = NULL, unique = FALSE, ...)

db_drop_table(con, table, force = FALSE, ...)

db_analyze(con, table, ...)

db_explain(con, sql, ...)

db_query_fields(con, sql, ...)

db_query_rows(con, sql, ...)

sql_select(con, select, from, where = NULL, group_by = NULL,
  having = NULL, order_by = NULL, limit = NULL, distinct = FALSE, ...)

sql_subquery(con, from, name = random_table_name(), ...)

sql_join(con, x, y, vars, type = "inner", by = NULL, ...)

sql_semi_join(con, x, y, anti = FALSE, by = NULL, ...)

sql_set_op(con, x, y, method)

sql_escape_string(con, x)

sql_escape_ident(con, x)


Arguments
con

A database connection.


table

A string, the table name.


fields

A list of fields, as in a data frame.



Is there any way that you give me an example of how to use this in a working code?

--
Heberto del Rio
Mathematics and Computer Science
Barry University


On February 21, 2018 at 6:42:15 PM, Edgar Ruiz (notifications@github.com<mailto:notifications@github.com>) wrote:

Oh, that function is in dbplyr

**Comment by edgararuiz** [comment]
Right but the `sql_subquery.SQLiteConnection` is a function inside `dbplyr: https://github.com/tidyverse/dbplyr/blob/ac41874070e063b939726cbb999ebe800dab2a27/R/db-
sqlite.r#L51-L61

**Comment by hdelrio** [comment]
I finally was able to do what you suggested but it did not fix the problem:

> library(dbplyr)
> library(dplyr)

Attaching package: ‘dplyr’

The following objects are masked from ‘package:dbplyr’:

    ident, sql

The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

> library(odbc)
Warning message:
package ‘odbc’ was built under R version 3.4.3
> con <- dbConnect(odbc::odbc(), "NFL")
> sql_subquery.SQLite = dbplyr:::sql_subquery.SQLiteConnection
> offense_db = tbl(con,'offense')
> player_db = tbl(con, 'player')
> offense_db %>% inner_join(player_db %>% filter(pos1=='QB'), by='player')
Error in new_result(connection@ptr, statement) :
  nanodbc/nanodbc.cpp:1344: HY000: [SQLite]no such column: TBL_RIGHT.fname (1)
> offense_db = tbl(con,'offense')
> player_db = tbl(con, 'player')
> offense_db %>% inner_join(player_db %>% filter(pos1=='QB'), by='player') %>% show_query()
<SQL>
SELECT "TBL_LEFT"."uid" AS "uid", "TBL_LEFT"."gid" AS "gid", "TBL_LEFT"."player" AS "player", "TBL_LEFT"."pa" AS "pa", "TBL_LEFT"."pc" AS "pc", "TBL_LEFT"."py" AS "py", "TBL_LEFT"."ints" AS "ints", "TBL_LEFT"."tdp" AS "tdp", "TBL_LEFT"."ra" AS "ra", "TBL_LEFT"."sra" AS "sra", "TBL_LEFT"."ry" AS "ry", "TBL_LEFT"."tdr" AS "tdr", "TBL_LEFT"."trg" AS "trg", "TBL_LEFT"."rec" AS "rec", "TBL_LEFT"."recy" AS "recy", "TBL_LEFT"."tdrec" AS "tdrec", "TBL_LEFT"."ret" AS "ret", "TBL_LEFT"."rety" AS "rety", "TBL_LEFT"."tdret" AS "tdret", "TBL_LEFT"."fuml" AS "fuml", "TBL_LEFT"."peny" AS "peny", "TBL_LEFT"."conv" AS "conv", "TBL_LEFT"."fp" AS "fp", "TBL_LEFT"."fp2" AS "fp2", "TBL_LEFT"."fp3" AS "fp3", "TBL_LEFT"."game" AS "game", "TBL_LEFT"."seas" AS "seas", "TBL_LEFT"."year" AS "year", "TBL_LEFT"."team" AS "team", "TBL_LEFT"."posd" AS "posd.x", "TBL_LEFT"."jnum" AS "jnum.x", "TBL_LEFT"."dcp" AS "dcp.x", "TBL_LEFT"."nflid" AS "nflid.x", "TBL_RIGHT"."fname" AS "fname", "TBL_RIGHT"."lname" AS "lname", "TBL_RIGHT"."pname" AS "pname", "TBL_RIGHT"."pos1" AS "pos1", "TBL_RIGHT"."pos2" AS "pos2", "TBL_RIGHT"."height" AS "height", "TBL_RIGHT"."weight" AS "weight", "TBL_RIGHT"."dob" AS "dob", "TBL_RIGHT"."forty" AS "forty", "TBL_RIGHT"."bench" AS "bench", "TBL_RIGHT"."vertical" AS "vertical", "TBL_RIGHT"."broad" AS "broad", "TBL_RIGHT"."shuttle" AS "shuttle", "TBL_RIGHT"."cone" AS "cone", "TBL_RIGHT"."arm" AS "arm", "TBL_RIGHT"."hand" AS "hand", "TBL_RIGHT"."dpos" AS "dpos", "TBL_RIGHT"."col" AS "col", "TBL_RIGHT"."dv" AS "dv", "TBL_RIGHT"."start" AS "start", "TBL_RIGHT"."cteam" AS "cteam", "TBL_RIGHT"."posd" AS "posd.y", "TBL_RIGHT"."jnum" AS "jnum.y", "TBL_RIGHT"."dcp" AS "dcp.y", "TBL_RIGHT"."nflid" AS "nflid.y", "TBL_RIGHT"."pimg" AS "pimg"
  FROM "offense" AS "TBL_LEFT"
  INNER JOIN (SELECT *
FROM "player"
WHERE ("pos1" = 'QB')) AS "TBL_RIGHT"
  ON ("TBL_LEFT"."player" = "TBL_RIGHT"."player")

--
Heberto del Rio
Mathematics and Computer Science
Barry University


On February 21, 2018 at 6:55:12 PM, Edgar Ruiz (notifications@github.com<mailto:notifications@github.com>) wrote:

Right but the sql_subquery.SQLiteConnection is a function inside `dbplyr: https://github.com/tidyverse/dbplyr/blob/ac41874070e063b939726cbb999ebe800dab2a27/R/db-<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Ftidyverse%2Fdbplyr%2Fblob%2Fac41874070e063b939726cbb999ebe800dab2a27%2FR%2Fdb-&data=02%7C01%7Chdelrio%40barry.edu%7C0a12c722311a4a9f849808d57986899b%7C7549241466ff4a7ca7f16c6787413d30%7C0%7C0%7C636548541119459953&sdata=xZQcEf%2Fb7nMy3jX7B4gdPHP9f0lUGNbOP8DvLAUtPDk%3D&reserved=0>
sqlite.r#L51-L61

**Comment by edgararuiz** [comment]
Oh, I'm sorry, I thought the objective was to make sure the the queries were aliased using **AS** as you mentioned in your initial post:

WHERE ("pos1" = "QB")) AS "TBL_RIGHT"

ON (TBL_LEFT.player = TBL_RIGHT.player)

The problem is simple to solve as can be seen from the above two examples, the bolded part, I have to change the quotes in the bolded part of the second example so the markdown would look similar in both cases

**Comment by hdelrio** [comment]
That is what I thought, but apparently that is not the problem! The problem is that some queries do work indistinctly of the connection (ODBC or DBI) using dplyr as front-end, but when I try to use an inner_join it fails when ODBC is being used as opposed when DBI is used, when I looked at the SQL query generated I thought that in the ODBC case was ill constructed (I noticed the missing AS at the right table) as opposed to the time when DBI is used.

If I where have written the sql query by hand I would have done it like the DBI case, that is why when I finally manage to implement your suggestion and looked at the generated SQL query I got surprised that it was correct but still I end up having an error

Any suggestions?

--
Heberto del Rio
Mathematics and Computer Science
Barry University


On February 21, 2018 at 8:03:17 PM, Edgar Ruiz (notifications@github.com<mailto:notifications@github.com>) wrote:

Oh, I'm sorry, I thought the objective was to make sure the the queries were aliased using AS as you mentioned in your initial post:

WHERE ("pos1" = "QB")) AS "TBL_RIGHT"

ON (TBL_LEFT.player = TBL_RIGHT.player)

The problem is simple to solve as can be seen from the above two examples, the bolded part, I have to change the quotes in the bolded part of the second example so the markdown would look similar in both cases
